pkgs/tools/system/gdu/default.nix

```
@@ -1,32 +1,41 @@
 { lib
+, stdenv
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
 }:
 
 buildGoModule rec {
   pname = "gdu";
-  version = "4.3.2";
+  version = "4.6.1";
 
   src = fetchFromGitHub {
     owner = "dundee";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-m4J797bmQzKuyA01JgDLVlf+PbXDVXWtYbID/0QVLxE=";
+    sha256 = "14r0plyzb4y38x2ipaimv57alr9sbk7xqc5j58xvfj43196yc4r7";
   };
 
-  vendorSha256 = "sha256-kIMd0xzQ+c+jCpX2+qdD/GcFEirR15PMInbEV184EBU=";
+  vendorSha256 = "058h71gmgi3n4b697myi5890arzw8fkzmxlm1aiwzyfh3k9iv0wh";
 
-  buildFlagsArray = [ "-ldflags=-s -w -X github.com/dundee/gdu/build.Version=${version}" ];
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X github.com/dundee/gdu/build.Version=${version}"
+  ];
 
   nativeBuildInputs = [ installShellFiles ];
 
+  postPatch = ''
+    substituteInPlace cmd/run_test.go --replace "development" "${version}"
+  '';
+
   postInstall = ''
     installManPage gdu.1
   '';
 
-  # tests fail if the version is set
-  doCheck = false;
+  doCheck = !stdenv.isAarch64;
```

We can keep the test disabled for darwin.

```suggestion
  doCheck = !(stdenv.isAarch64 || stdenv.isDarwin);
```

---

Done, thanks.